<Type Name="XamlSchemaException" FullName="System.Xaml.XamlSchemaException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="c0e3c0553d95066cc83a86ae69345ca7e0056855" /><Meta Name="ms.sourcegitcommit" Value="055a4a82a0b08bfbdc21bd1347fb71f7fe2c099e" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="pl-PL" /><Meta Name="ms.lasthandoff" Value="08/15/2019" /><Meta Name="ms.locfileid" Value="69415945" /></Metadata><TypeSignature Language="C#" Value="public class XamlSchemaException : System.Xaml.XamlException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit XamlSchemaException extends System.Xaml.XamlException" />
  <TypeSignature Language="DocId" Value="T:System.Xaml.XamlSchemaException" />
  <TypeSignature Language="VB.NET" Value="Public Class XamlSchemaException&#xA;Inherits XamlException" />
  <TypeSignature Language="C++ CLI" Value="public ref class XamlSchemaException : System::Xaml::XamlException" />
  <TypeSignature Language="F#" Value="type XamlSchemaException = class&#xA;    inherit XamlException" />
  <AssemblyInfo>
    <AssemblyName>System.Xaml</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Xaml.XamlException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="bae9a-101">Wyjątek, który jest generowany, gdy system powiązań lub inny system reprezentacji schematu dla języka XAML zgłasza wyjątek do kontekstu schematu.</span><span class="sxs-lookup"><span data-stu-id="bae9a-101">The exception that is thrown when a binding system or another schema representation system for XAML reports an exception to the schema context.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="bae9a-102">Jest zgłaszany przez kilka przypadków w klasach obsługi powiązań CLR, które udostępniają kontekst schematu dla .NET Framework usług XAML<xref:System.Xaml.XamlXmlReader>( <xref:System.Xaml.XamlObjectWriter>, lub <xref:System.Xaml.XamlServices>). <xref:System.Xaml.XamlSchemaException></span><span class="sxs-lookup"><span data-stu-id="bae9a-102">The <xref:System.Xaml.XamlSchemaException> is thrown by several cases in the CLR binding support classes that provide the schema context for .NET Framework XAML Services (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, or <xref:System.Xaml.XamlServices>).</span></span> <span data-ttu-id="bae9a-103">Jest zwykle przenoszona jako wewnętrzny wyjątek dla innych wyjątków, takich <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>jak właściwość-true dla a <xref:System.Xaml.XamlObjectWriterException>, <xref:System.Xaml.XamlSchemaException> dlatego powinna być nierzadko, chyba że implementujesz składnik, taki jak <xref:System.Xaml.XamlSchemaException> wyspecjalizowany moduł zapisujący obiektów.</span><span class="sxs-lookup"><span data-stu-id="bae9a-103">A <xref:System.Xaml.XamlSchemaException> is typically passed as the inner exception for other exceptions, such as the <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>-true property for a <xref:System.Xaml.XamlObjectWriterException>, so it should be uncommon to catch a <xref:System.Xaml.XamlSchemaException> directly unless you are implementing a component such as a specialized object writer.</span></span>  
  
 ]]></format>
    </remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="bae9a-104">Inicjuje nowe wystąpienie klasy <see cref="T:System.Xaml.XamlSchemaException" /> klasy.</span><span class="sxs-lookup"><span data-stu-id="bae9a-104">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="bae9a-105">Inicjuje nowe wystąpienie <see cref="T:System.Xaml.XamlSchemaException" /> klasy z komunikatem dostarczonym przez system, który opisuje błąd.</span><span class="sxs-lookup"><span data-stu-id="bae9a-105">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a system-supplied message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="bae9a-106">Komunikat, który opisuje wyjątek.</span><span class="sxs-lookup"><span data-stu-id="bae9a-106">The message that describes the exception.</span></span> <span data-ttu-id="bae9a-107">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span><span class="sxs-lookup"><span data-stu-id="bae9a-107">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <summary><span data-ttu-id="bae9a-108">Inicjuje nowe wystąpienie <see cref="T:System.Xaml.XamlSchemaException" /> klasy z określonym komunikatem opisującym błąd.</span><span class="sxs-lookup"><span data-stu-id="bae9a-108">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; XamlSchemaException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="bae9a-109">Obiekt zawierający dane serializowanego obiektu.</span><span class="sxs-lookup"><span data-stu-id="bae9a-109">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="bae9a-110">Informacje kontekstowe o źródłowego lub miejscu docelowym.</span><span class="sxs-lookup"><span data-stu-id="bae9a-110">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="bae9a-111">Inicjuje nowe wystąpienie klasy <see cref="T:System.Xaml.XamlSchemaException" /> klasy z serializowanych danych.</span><span class="sxs-lookup"><span data-stu-id="bae9a-111">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with serialized data.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string * Exception -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="bae9a-112">Komunikat, który opisuje wyjątek.</span><span class="sxs-lookup"><span data-stu-id="bae9a-112">The message that describes the exception.</span></span> <span data-ttu-id="bae9a-113">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span><span class="sxs-lookup"><span data-stu-id="bae9a-113">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <param name="innerException"><span data-ttu-id="bae9a-114">Wyjątek, który jest przyczyną bieżącego wyjątku.</span><span class="sxs-lookup"><span data-stu-id="bae9a-114">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="bae9a-115">Jeśli parametr nie <see langword="null" />jest, bieżący <see langword="catch" /> wyjątek jest wywoływany w bloku, który obsługuje wyjątek wewnętrzny. <paramref name="innerException" /></span><span class="sxs-lookup"><span data-stu-id="bae9a-115">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="bae9a-116">Inicjuje nowe wystąpienie klasy <see cref="T:System.Xaml.XamlSchemaException" /> klasy przy użyciu określonego komunikatu o błędzie i odwołania do wewnętrznego wyjątku, który jest przyczyną tego wyjątku.</span><span class="sxs-lookup"><span data-stu-id="bae9a-116">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="bae9a-117">Wyjątek generowany jako bezpośredni wynik poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A> właściwości.</span><span class="sxs-lookup"><span data-stu-id="bae9a-117">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="bae9a-118">Właściwość zwraca tę samą wartość, która jest przenoszona do konstruktora, lub `null` Jeśli <xref:System.Exception.InnerException%2A> właściwość nie poda wewnętrznej wartości wyjątku do konstruktora. <xref:System.Exception.InnerException%2A></span><span class="sxs-lookup"><span data-stu-id="bae9a-118">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>
